### DISCUSSION TITLE
发现MaaFramework的Pipeline协议更新了字段，之前用项目模板创建的项目需要重新创建吗

### DISCUSSION THREAD

**Discussion by Soramii** [MaaXYZ/MaaFramework#396] [Q&A] [answered]
还是只要把MaaFramework的Release重新解压到deps替换就行了

**Comment by MistEO** [comment, marked as answer]
deps 更新一下，再参考这个改一下 pipeline

![image](https://github.com/user-attachments/assets/5df3f997-bbae-4d67-b214-f54daf26831d)
